config/stack_mac.yaml

```diff
@@ -51,6 +51,7 @@ netcdf:
   build: YES
   shared: YES
   enable_pnetcdf: NO
+  build_netcdf_cxx: YES
```

Comment:
This needs to be changed to `disable_cxx`
Also in the Known issues of the `README.md`

Reply:
Also, since `YES` is the default, we don't really need to set it explicitly to YES.  It is only when one wishes to disable it, it needs to be told to do so.

Reply:
In other words, remove this line.